index.html

```diff
@@ -181,6 +181,29 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+    var fireballSize = 22;
+
+    var getFireballSpeed = function (left) {
+      return left ? 5 : 2;
+    };
+
+    var wizardSpeed = 3;
+
+    var wizardWidth  = 70;
+
+    var getWizardHeight = function (wizardWidth) {
+      return 1.337 * wizardWidth;
+    };
+
+    var getWizardX = function (width) {
+      return (width - wizardWidth/2) * 0.5;
+    };
+
+    var getWizardY = function (height) {
+      return height * 0.33 + (1.337 * wizardWidth)/2;
```

Comment:
Можно было использовать уже определенную функцию для вычисления высоты мага:
```suggestion
      return height * 0.33 + getWizardHeight() / 2;
```